specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -858,6 +862,20 @@ components:
           type: "array"
           items:
             $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/schemas/HalLink"
+    Koppelingswijze_enum:
+      type: "string"
+      description: "Koppelingswijze is de beschrijving van de manier waarop de koppeling tussen het Kadastraalobject (Perceel of Appartementsrecht) met ObjectlocatieBinnenland bepaald is. De koppeling kan administratief bepaald zijn op basis van overeenkomstige adresgegevens en of geometrisch bepaald zijn, op basis van overeenkomstige geometrie."
+      enum:
+      - "Administratief"
+      - "Geometrisch"
+      - "Administratief en Geometrisch"
+    LocatieKadastraalObject:
+      type: "object"
+      properties:
+        nummeraanduidingIdentificatie:
+          type: "string"
+        koppelingswijze:
+          $ref: "#/components/schemas/Koppelingswijze_enum"
```

Comment:
In het IMKAD is koppelingswijze een waardelijst. @CathyDingemanse moet dit een enumeratie zijn? (zodat gebruikers code kunnen schrijven voor automatisch afhandelen specifieke waarden hiervoor?)

Wanneer dit vooral bedoeld is voor een eindgebruiker (mens) om te lezen, dan zou ik het gewoon als waardelijst definiëren.

Reply:
- Kan koppelingswijze worden uitgebreid?
- Kunnen de waardes anders gaan heten?
- Is de code van 'administratief', 'geometrisch' en 'administratief en geometrisch' ongelijk aan resp. 0, 1, 2
Als één van de drie punten met ja wordt beantwoord, dan is waardelijst een betere keuze

Reply:
in de waardelijst zijn er de codes "ADM_GEO", "ADM" en "GEO"